tests/phpunit/Helpers/UPE_Test_Helper.php

```
@@ -14,29 +14,21 @@
-	public function enable_upe_feature_flag() {
-		// Force the UPE feature flag on.
-		add_filter(
-			'pre_option__wcstripe_feature_upe',
-			function () {
-				return 'yes';
-			}
-		);
+	public function enable_upe_feature_flag(): void {
 		WC_Stripe_Helper::delete_main_stripe_settings();
 		$this->reload_payment_gateways();
 	}
```

PHPMD (2.15.0)

[error] The method enable_upe_feature_flag is not named in camelCase. (undefined)

(CamelCaseMethodName)